doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -430,5 +430,274 @@ gls{SoMoX}
 .
 \end_layout
 
+\begin_layout Subsection
+GUI model
+\end_layout
+
+\begin_layout Standard
+The user has multiple options to launch the analysis:
+\end_layout
+
+\begin_layout Enumerate
+To analyse the whole project, there is an entry 
+\begin_inset Quotes eld
+\end_inset
+
+Analyse with Beagle
+\begin_inset Quotes erd
+\end_inset
+
+ in the context menu of the 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{.repository}
+\end_layout
+
+\end_inset
+
+ or 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{.repository_diagram}
+\end_layout
+
+\end_inset
+
+ file in eclipse.
+\end_layout
+
+\begin_layout Enumerate
+To analyse a single component, there is an entry 
+\begin_inset Quotes eld
+\end_inset
+
+Analyse with Beagle
+\begin_inset Quotes erd
+\end_inset
+
+ in the context menu of each 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{component}
+\end_layout
+
+\end_inset
+
+ in the repository diagram, where an analysis with Beagle is possible.
+ Otherwise the context menu entry will be shown greyed and a description
+ why the analysis is not possible appears, when the users tries to start
+ the analysis.
+\end_layout
+
+\begin_layout Enumerate
+To analyse a single 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{internal action}
+\end_layout
+
+\end_inset
+
+, there is an entry 
+\begin_inset Quotes eld
+\end_inset
+
+Analyse with Beagle
+\begin_inset Quotes erd
+\end_inset
+
+ in the context menu of each 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{internal action}
+\end_layout
+
+\end_inset
+
+ in the 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SEFF}
+\end_layout
+
+\end_inset
+
+ diagram.
+\end_layout
+
+\begin_layout Enumerate
+If 
+\begin_inset CommandInset ref
+LatexCommand ref
+reference "/OQ100/"
+
+\end_inset
+
+ is realized, the user has the option, when launching 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SoMoX}
+\end_layout
+
+\end_inset
+
+ to automatically launch the analysis with Beagle after 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SoMoX}
+\end_layout
+
+\end_inset
+
+ has finished.
+\end_layout
+
+\begin_layout Standard
+When the users launches the analysis by a context menu (option 1, 2 or 3),
+ he gets an window where he can change some settings for Beagle:
```

Comment:
But it's another window there, isn't it?
I thought we just add some GUI elements to a SoMoX window...